luigi/parameter.py

```diff
@@ -497,6 +506,19 @@ def normalize(self, value):
         return datetime.date(year=value.year - delta, month=1, day=1)
 
 
+class PastDateParameter(DateParameter):
+    """
+    caveat: if your host default timezone is different from your `business` timezone
+    that might not work for you
+    """
+
+    def normalize(self, value):
+        if value >= datetime.date.today():
```

Comment:
It feels a little weird to evaluate "today" using system time. (Not saying this makes sense, but) suppose your system time is local but you eval based on UTC or visa versa.

Reply:
@dlstadther
okay, that's fair
updated pr with possibility of choosing timezone - local or utc